validation/src/test/java/org/craftercms/commons/validation/validators/impl/ContentNewPathValidatorTest.java

```diff
@@ -101,6 +101,13 @@ public void testHash() {
         assertRejected("/site/website/folder#hash/to/content/index.xml");
     }
 
+    @Test
+    public void testInvalidPathTraversal() {
+        assertRejected("../../site/website");
```

Comment:
@phuongnq please consider these scenarios as well: https://github.com/craftercms/craftercms/issues/7047

Reply:
Updated to allow curly braces